appstudio-controller/controllers/appstudio.redhat.com/devsandbox_deployment_controller.go

```diff
@@ -195,11 +201,6 @@ func CreateDeploymentTargetForSpaceRequest(ctx context.Context, client client.Cl
 		deploymentTarget.Labels[applicationv1alpha1.AnnTargetProvisioner] = dtc.Annotations[applicationv1alpha1.AnnTargetProvisioner]
 	}
 
```

Comment:
I've removed the controller ownership, because it interferes with some of the expected behaviours defined in ADR-8. When I originally proposed controller ownership, I did not fully grok the expected behaviour.